cypress/integration/chi-vue/search-input.spec.js

```diff
@@ -0,0 +1,207 @@
+const SEARCH_INPUTS_DATA_CY = {
+  BASE: '[data-cy="base-search-input-test"]',
+  DISABLED: '[data-cy="disabled-search-input-test"]',
+  ATTRS: '[data-cy="attrs-search-input-test"]',
+  DELETE: '[data-cy="delete-value-search-input-test"]',
+  EVENTS: '[data-cy="chi-events-trigger-search-input-test"]'
+};
+const ELEMENTS = {
+  BUTTON: 'button',
+  DIV: 'div',
+  ICON: 'i',
+  INPUT: 'input'
+};
+const COMPONENT_CLASSES = {
+  CLOSE: '-close',
+  PORTAL: '-icon--primary',
+  SEARCH: '-flat'
+};
+const EVENT_EXAMPLES = {
+  BLUR: 'eventExampleChiBlur',
+  CHANGE: 'eventExampleChiChange',
+  CLEAN: 'eventExampleChiClean',
+  FOCUS: 'eventExampleChiFocus',
+  INPUT: 'eventExampleChiInput',
+  SEARCH: 'eventExampleChiSearch',
+};
+const EVENTS = {
+  BLUR: 'chiBlur',
+  CHANGE: 'chiChange',
+  CLEAN: 'chiClean',
+  FOCUS: 'chiFocus',
+  INPUT: 'chiInput',
+  SEARCH: 'chiSearch',
+};
+
+const checkSizeClass = (el, value) => {
+  cy.get(el).find(`${ELEMENTS.INPUT}`).should('have.class', value);
+};
+
+const checkAttr = (el, sel, attr, value) => {
```

Comment:
The element should be the one you check the attribute for. What you are doing here is receive 1) A parent element; 2) A selector by which you find the target element. I suggest you receive the target element directly from outside.